app.js

```diff
@@ -1,16 +1,19 @@
 const express = require('express')
 const {getTopics} = require('./controllers/topics.controller.js')
 const {getEndpoints} = require('./controllers/endpoints.controller.js')
+const {getArticleById} = require('./controllers/articles.controller.js')
 
 
 // Set up. 
 const app = express()
 app.use(express.json())
```

Comment:
You don't need this yet (app.use(express.json())) so please remove it until you do.